src/Nethermind/Nethermind.Merge.Plugin/Data/GetBlobsV1Result.cs

```diff
@@ -0,0 +1,12 @@
+// SPDX-FileCopyrightText: 2024 Demerzel Solutions Limited
+// SPDX-License-Identifier: LGPL-3.0-only
+
+using System.Collections.Generic;
+using Nethermind.TxPool;
+
+namespace Nethermind.Merge.Plugin.Data;
+
+public class GetBlobsV1Result(IEnumerable<BlobAndProofV1?> blobsAndProofs)
+{
+    public readonly IEnumerable<BlobAndProofV1?> BlobsAndProofs = blobsAndProofs;
```

Comment:
Nitpick: If no specific reason (I don't see one), this member should be a get-only property instead of a read-only field, according to Microsoft's guidelines.

```suggestion
    public IEnumerable<BlobAndProofV1?> BlobsAndProofs { get; } = blobsAndProofs;
```